.github/workflows/qiita-ci.yml

```diff
@@ -169,17 +169,20 @@ jobs:
           conda deactivate
 
           echo "8. Setting up SSH"
-          ssh-keygen -t rsa -b 4096 -N '' -f $PWD/qiita_ware/test/test_data/test_key
+          ssh-keygen -t ed25519 -a 200 -N '' -f $PWD/qiita_ware/test/test_data/test_key
           mkdir ~/.ssh/
           cp $PWD/qiita_ware/test/test_data/test_key* ~/.ssh/
           cat ~/.ssh/test_key.pub > ~/.ssh/authorized_keys
+          cat ~/.ssh/test_key.pub > ~/.ssh/authorized_keys2
           chmod 600 $PWD/qiita_ware/test/test_data/test_key*
           chmod 600 ~/.ssh/*
           chmod 700 ~/.ssh/
           echo "Connecting as $USER@localhost"
           # this line (and the -o StrictHostKeyChecking=no) is so the server
           # is added to the list of known servers
-          scp -O -o StrictHostKeyChecking=no -i $PWD/qiita_ware/test/test_data/test_key $USER@localhost:/home/runner/work/qiita/qiita/qiita_ware/test/test_data/random_key /home/runner/work/qiita/qiita/qiita_ware/test/test_data/random_key_copy_1
+
+          # 05/22/25: commenting this line out as github actions is not allowing this step
+          # scp -O -o StrictHostKeyChecking=no -i $PWD/qiita_ware/test/test_data/test_key $USER@localhost:/home/runner/work/qiita/qiita/qiita_ware/test/test_data/random_key /home/runner/work/qiita/qiita/qiita_ware/test/test_data/random_key_copy_1
```

Comment:
What is the effect of commenting this out?  I see the comment above that says "this line is so the server is added to the list of known servers", so I assume that now it isn't added to the list of known servers :) ... but what effect does that have on behavior?

Reply:
This is the main culprit of the failings tests as it sets all the internal parameters for SCP to work by doing a single file scp; however, in all the builds it failed.